misc/services/mobile-webui/mobile-webui-frontend/src/actions/PickingActions.js

```diff
@@ -1,4 +1,4 @@
-import { UPDATE_PICKING_STEP_QTY } from '../constants/PickingActionTypes';
+import { UPDATE_PICKING_STEP_QTY, UPDATE_ALT_PICKING_STEP_QTY } from '../constants/PickingActionTypes';
```

Comment:
IMHO UPDATE_PICKING_STEP_QTY and UPDATE_ALT_PICKING_STEP_QTY actions shall be the same.
The only diff is that u have to pass the `alternativeId` around.

Reply:
Done